integration/pom.xml

```diff
@@ -0,0 +1,108 @@
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>beadledom-integration</artifactId>
+  <packaging>pom</packaging>
+  <name>Beadledom Integration</name>
+
+  <parent>
+    <artifactId>beadledom-parent</artifactId>
+    <groupId>com.cerner.beadledom</groupId>
+    <version>3.3-SNAPSHOT</version>
+  </parent>
+
+  <modules>
+    <module>api</module>
+    <module>service</module>
+    <module>client</module>
+    <module>test</module>
+  </modules>
+
+  <properties>
+    <jetty.version>9.4.18.v20190429</jetty.version>
+    <slf4j.version>1.7.25</slf4j.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.cerner.beadledom</groupId>
+        <artifactId>beadledom-integration-client</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.cerner.beadledom</groupId>
+        <artifactId>beadledom-integration-api</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.cerner.beadledom</groupId>
+        <artifactId>beadledom-integration-service</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.cerner.beadledom</groupId>
+        <artifactId>beadledom-resteasy</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.cerner.beadledom</groupId>
+        <artifactId>beadledom-swagger2</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-client</artifactId>
+        <version>${resteasy.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-guice</artifactId>
+        <version>${resteasy.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-jaxrs</artifactId>
+        <version>${resteasy.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.netflix.governator</groupId>
+        <artifactId>governator</artifactId>
+        <version>1.17.4</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.6.0</version>
+        <configuration>
+          <source>${java.version}</source>
+          <target>${java.version}</target>
+        </configuration>
+      </plugin>
+    </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-war-plugin</artifactId>
+          <version>3.0.0</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.tomcat.maven</groupId>
```

Comment:
Is tomcat being used?

Reply:
Nope, I removed it: https://github.com/cerner/beadledom/pull/149/commits/5f16000751c242d73a0ec08af64c59f7b354e354